scripts/release.sh

```diff
@@ -98,9 +114,24 @@ yarn run update-milestones ${npm_package_version}
 step "create-github-release"
 yarn run create-github-release ${npm_package_version}
 
-step "manual step: Update the documentation"
-echo "You can use the 'copy-ra-oss-docs.sh' script if you have it"
-echo "Press Enter when this is done"
-read
+if [ -d $RA_DOC_PATH ]; then
+    step "Update the documentation"
+    # ${npm_package_version%.*} extract the major.minor version
+    RA_DOC_PATH="$RA_DOC_PATH" VERSION="${npm_package_version%.*}" ./scripts/copy-ra-oss-docs.sh
+    # Set the latest version in the versions.yml file
+    echo "Update the latest version in the versions.yml file"
+    sed -i "/^\(- latest\).*/s//\1 \($npm_package_version\)/" $RA_DOC_PATH/_data/versions.yml
+    if [ "${npm_current_package_version%.*}" == "${npm_package_version%.*}" ]; then
+        echo "Add the previous minor version to the list of versions in the versions.yml file"
+        # Add the previous minor version to the list of versions in the versions.yml file
+        sed -i "/^\(- latest.*\)/s//\1 \n- \"${npm_package_version%.*}\"/" $RA_DOC_PATH/_data/versions.yml
+    fi
```

Comment:
For minor releases: You could add an `echo` statement to remind the user they need to update the `OldVersions.md` file.

Besides, for all releases: You could also add an `echo` statement to remind the user they still need to `git commit` and `git push` themselves.